localisation/new_loc_l_english.yml

```diff
@@ -308,7 +308,7 @@
  flavor_jnk.1001.t: "Uncontrollable Grudge"
  flavor_jnk.1001.desc: "The thing we feared most has happened: Junko's simmering anger and grudge has reached what she can no longer contain! She has begun to run amok, perceiving everything in front of her as chang'e and unleashing her fury all around. Her court and capital facilities have begun to burn, and her aides who attempted to calm her down have been consumed by the flames of her rage! Her anger has been pent up for so many centuries, so it will take quite some time for her to recover from her rampage and regain her senses..."
  flavor_jnk.1001.a: "All we can do is endure her wrath for now..."
- flavor_jnk.1001.a.tt: "Have to wait §Y5§! years before §YJunko§! comes to her senses and returns to our court..."
+ flavor_jnk.1001.a.tt: "§YCall our Ruler back§! decision will be §Rdisabled§! for §Y5 years!§!\n\nWe Have to wait 5 years before §YJunko§! recover her senses and returns to our court. After that, §YCall our Ruler back§! decision will be §Genabled§! again."
```

Comment:
In the second sentence would change some stuff to make it sound more natural (would suggest changing "We Have" -> "We have", and "Junko recover her senses" -> "Junko recovers her senses"), but elsewhere is fine.